dashed/assets/javascripts/featured.js

```diff
@@ -1,6 +1,6 @@
 var $ = window.$ = require('jquery');
 var jQuery = window.jQuery = $;
-var px = require('./modules/dashed.js');
+require('./modules/dashed.js');
```

Comment:
It was to get `dashed.css` incorporate, I also figured I might need things from `px` eventually. 

Now I'm realizing `px` doesn't make sense anymore as it was a short for panoramix...